lib/features/refactorings/RefactoringsTracking.js

```diff
@@ -0,0 +1,24 @@
+export default class ContextPadTracking {
+  constructor(eventBus, bpmnJSTracking) {
+    this._bpmnJSTracking = bpmnJSTracking;
+    this._eventBus = eventBus;
+
+    this._eventBus.on('refactorings.execute', this.trackRefactoring.bind(this));
+  }
+
+  trackRefactoring(event) {
+    const { refactoring } = event;
+
+    this._bpmnJSTracking.track({
+      name: 'refactorings.execute',
+      data: {
+        refactoring
```

Comment:
Quick-check: Can this leak personally identifyable information?

Reply:
We recently made a change to [sanatize all events](https://github.com/bpmn-io/bpmn-js-tracking/blob/main/lib/BpmnJSTracking.js#L70) so we don't have to do it manually in each handler.